src/main/events/library/open-game.ts

```diff
@@ -2,18 +2,31 @@ import { gameRepository } from "@main/repository";
 
 import { registerEvent } from "../register-event";
 import { shell } from "electron";
+import { spawn } from "child_process";
 import { parseExecutablePath } from "../helpers/parse-executable-path";
+import { parseLaunchOptions } from "../helpers/parse-launch-options";
 
 const openGame = async (
   _event: Electron.IpcMainInvokeEvent,
   gameId: number,
-  executablePath: string
+  executablePath: string,
+  launchOptions: string | null
 ) => {
   const parsedPath = parseExecutablePath(executablePath);
+  const parsedParams = parseLaunchOptions(launchOptions);
 
-  await gameRepository.update({ id: gameId }, { executablePath: parsedPath });
+  await gameRepository.update(
+    { id: gameId },
+    { executablePath: parsedPath, launchOptions }
+  );
 
-  shell.openPath(parsedPath);
+  if (process.platform === "linux" || process.platform === "darwin") {
+    shell.openPath(parsedPath);
+  }
+
+  if (process.platform === "win32") {
+    spawn(parsedPath, parsedParams, { shell: false, detached: true });
+  }
```

Comment:
this feature would probably be useful for other SO as well.
but we leave it like this until we test in Linux/Mac

Reply:
I chose to use this if statement and keep Linux/macOS as it was because I noticed that on Linux, Hydra only supports games via Wine, and on macOS, there is no Hydra release. I did a small test on Ubuntu 22.04 LTS and noticed that the `spawn` behavior is different from `shell.openPath`, but I couldn't configure Wine to test it